test/functional/wallet_migration.py

```diff
@@ -445,6 +445,39 @@ def test_no_privkeys(self):
         # After migrating, the "keypool" is empty
         assert_raises_rpc_error(-4, "Error: This wallet has no available keys", watchonly1.getnewaddress)
 
+        # Migrating a watch-only blank empty wallet (with no pubkeys)
+        self.log.info("Test migration of a pure watchonly blank empty wallet (with no pubkeys)")
+        watchonly2 = self.create_legacy_wallet("watchonly2", disable_private_keys=True, blank=True)
```

Comment:
the `blank=True` flag is not needed when you disable private keys.

Reply:
But wanted to make it explicit that as the wallet is empty, not imported address, the flag will remain the same after migration.